DANS-KNAW/easy-deposit-agreement-creator
# EASY-2209: v4.0 gdpr

fixes EASY-2209: v4.0 gdpr

## Squash merge comment
* new version of the depositor agreement
* uses only few emd-fields
* no longer needs a human readable version for the audience
* no longer needs fsrdb
* the font has changed, otherwise printing the document failed
* added easy-licenses resources for appendix3

## Where should the reviewer @DANS-KNAW/easy start?

Check the results of `fc-list` on production, we need the DejaVu font.

## How should this be manually tested?

* See comment of https://drivenbydata.atlassian.net/browse/EASY-2420
* submit deposits for various document variations
  * with a date available of today or in the future (embargo clause in appendix 1)
  * with any of the possible licenses (name in appendix 1, content in appendix 3)
  * with/without privacy sensitive data (a/no appendix3?)
* log-in on easy as archivist, go to the administration tab for each of the ingested datasets
* examine the depositor agreement, pay special attention to unintentional wrapping of content

## related pull requests on github
also upgrade https://github.com/DANS-KNAW/easy-licenses/ in
* [x] https://github.com/DANS-KNAW/easy-deposit-ui
* [x] https://github.com/DANS-KNAW/easy-validate-dans-bag
* [x] https://github.com/DANS-KNAW/easy-stage-dataset
* [x] https://github.com/DANS-KNAW/easy-ingest-flow
* [x] https://github.com/DANS-KNAW/easy-split-multi-deposit
